src/stories/select/Select.tsx

```diff
@@ -233,6 +239,28 @@ export const Select = forwardRef(function <T = Record<string, any>>({
                     ))}
                 </RadioGroup>
             )}
+            {kind === 'card' && (
+                <RadioGroup ref={ref} as="div" className={styles.cardContainer} value={value} onChange={onChange}>
+                    {options.map((option) => (
+                        <RadioGroup.Option
+                            as="div"
+                            className={clsx(
+                                styles.cardOption,
+                            )}
+                            key={option.id}
+                            value={option.id}
+                            disabled={option.disabled || false}
+                            data-status={disabled ? 'disabled' : (status || 'default')}
+                        >
+                            <div className={styles.cardSurface}>
+                                <TextWhenString kind="paragraphXSmall">
```

Comment:
similar to feedback on `CardButton`, we should provide good default styling here while still allowing users to override it